MojioSDK/Models/WifiRadio.swift

```diff
@@ -10,18 +10,52 @@ import UIKit
 import ObjectMapper
 import RealmSwift
 
+public class TransactionState: NSObject {
+    public static let Pending = "Pending"
+    public static let Success = "Success"
+    public static let Failure = "Failure"
+}
+
+public class WifiRadioStatus: NSObject {
+    public static let Connected = "Connected"
+    public static let Roaming = "Roaming"
+    public static let Disconnected = "Disconnected"
+}
+
 public class WifiRadio: Object, Mappable  {
     
     public dynamic var TimeStamp : String? = nil
     public dynamic var SSID : String? = nil
     public dynamic var Password : String? = nil
     public dynamic var AllowRoaming : String? = nil
-    public dynamic var Status : String? = nil //['Connected', 'Roaming', 'Disconnected']
+    public dynamic var Status : String? = nil
     public dynamic var Strength : String? = nil
 
     public required convenience init?(_ map: Map) {
         self.init()
     }
+    
+    public func jsonDict(timeToLive: Int? = nil) -> NSDictionary {
+        let dictionary : NSMutableDictionary = NSMutableDictionary()
+        
+        if let ssid = self.SSID {
+            dictionary.setObject(ssid, forKey: "SSID")
+        }
+
+        if let password = self.Password {
+            dictionary.setObject(password, forKey: "Password")
+        }
+        
+        if let status = self.Status {
+            dictionary.setObject(status, forKey: "Status")
+        }
+        
```

Comment:
Add a comment somewhere around ttl (Time To Live). A lot of developers will not be familiar with the terminology